### PR TITLE
fix(*): remove unnecessary omitted field of MutationObserverOptions

### PR DESCRIPTION
**Pull request by manudeli**

### CODE REVIEW

File: packages/angular-query-experimental/src/types.ts [hunk 1]
```diff
@@ -137,7 +137,7 @@ export interface CreateMutationOptions<
   TContext = unknown,
 > extends Omit<
     MutationObserverOptions<TData, TError, TVariables, TContext>,
-    '_defaulted' | 'variables'
```
Comment (manudeli) [inline packages/angular-query-experimental/src/types.ts:140]: There is no 'variable' field to omit in MutationObserverOptions

**Review by manudeli — COMMENTED** [review]
I left my intention

File: packages/angular-query-experimental/src/types.ts (hunk 1 above)
Comment (arnoud-dv) [reply packages/angular-query-experimental/src/types.ts:140]: All adapters except Vue Query contain this type. So I think it should be changed for all of those?

Comment (manudeli) [reply packages/angular-query-experimental/src/types.ts:140]: Sure, thanks for your eagle eyes 👍

**Review by TkDodo — APPROVED** [review]

**Pull request merged**